transcom/mymove
# B-19887: Add pricing estimate column to MTO Service Items

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19887)

## Summary

In this story the pricing_estimate column is being added to mto_service_items table.


### How to test

1. Run the migration scripts
2. Check the mto_service_items table for the pricing_estimate column
